### PR TITLE
[CPU]MVN_block_layout_tails_process_on_SSE_fix

### PR DESCRIPTION
**Pull request by chenhu-wang**
### Details:
 - *logic correction for block layout tails process on SSE*

### CODE REVIEW

**Comment by chenhu-wang** [comment]
@xipingyan , could you please take a look and verify if fix issue: https://github.com/openvinotoolkit/openvino/issues/20623

**Comment by xipingyan** [comment]
> @xipingyan , could you please take a look and verify if fix issue: #20623

Verified it pass based on `sde -ivb`

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/mvn.cpp [hunk 1]
```diff
@@ -175,6 +175,32 @@ const auto Mvn4DStatic = ::testing::Combine(
...
+               ::testing::ValuesIn(emptyReductionAxes()),
+               ::testing::Values(false),
+               ::testing::ValuesIn(normalizeVariance),
+               ::testing::ValuesIn(epsilon())),
+       ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D)),
+       ::testing::ValuesIn(fusingParamsSetStaticShape),
+       ::testing::Values(ElementType::f32),
+       ::testing::Values(ElementType::f32),
+       ::testing::ValuesIn(additionalConfig()));
+
```
Comment (xipingyan) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/mvn.cpp:201]: `additionalConfig` contains 2 data type f32 and f16, just test f32 data type is enough if we only test SSE41,  is it right? @chenhu-wang

Comment (chenhu-wang) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/mvn.cpp:205]: yes, one precision could cover this fix, done.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**